include/trajopt/path/SwervePathBuilder.h

```diff
@@ -21,6 +21,10 @@ namespace trajopt {
  */
 class TRAJOPT_DLLEXPORT SwervePathBuilder {
  public:
+  /**
+   * Cancel all currently generating SwervePathBuilders.
+   */
+  void CancelAll();
```

Comment:
Put an empty line between functions so it's clear which doxygen comment belongs to which.